internal/api/keppel/manifests.go

```diff
@@ -102,12 +104,14 @@ func (a *API) handleGetManifests(w http.ResponseWriter, r *http.Request) {
 			result.IsTruncated = true
 			break
 		}
+		labelsJSON := []byte(dbManifest.LabelsJSON)
```

Comment:
This tripped me up when reading since there are two casts happening in succession here. This could use some clarification:
```suggestion
		labelsJSON := json.RawMessage([]byte(dbManifest.LabelsJSON))
```